third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -254,6 +254,12 @@ struct LoadOpConversion : public ConvertOpToLLVMPattern<triton::LoadOp>,
       }
 
       if (other) {
+        if (otherIsSplatConstInt) {
+          for (size_t s = valueElemNBits; s < movWidth; s += valueElemNBits) {
+            splatVal |= splatVal << valueElemNBits;
+          }
+        }
```

Comment:
nit: can we just make a different variable instead of updating `splatVal`?

Reply:
Done.